### PR TITLE
bump secretshare version to 1.5.0

### PR DESCRIPTION
**Pull request by horis233**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

1. Please confirm that if this PR changes any image versions, then that's the sole change this PR makes.

**Release note**:

```release-note

```

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -205,7 +204,7 @@ multiarch-image: $(CONFIG_DOCKER_TARGET)
 # Generate bundle manifests and metadata, then validate generated files.
 bundle: manifests
 	operator-sdk generate kustomize manifests -q
-	kustomize build config/manifests | operator-sdk generate bundle -q --overwrite --version $(OPERATOR_VERSION) $(BUNDLE_METADATA_OPTS)
+	kustomize build config/manifests | operator-sdk generate bundle -q --overwrite --version $(VERSION) $(BUNDLE_METADATA_OPTS)
```
Comment (Daniel-Fan) [inline Makefile:207]: Should we use `version` or `release-version` here?

File: Makefile [hunk 2]
```diff
@@ -79,7 +78,7 @@ endif
 include common/Makefile.common.mk
 
 # Default bundle image tag
-BUNDLE_IMG ?= ibm-secretshare-operator-bundle:$(OPERATOR_VERSION)
+BUNDLE_IMG ?= ibm-secretshare-operator-bundle:$(VERSION)
```
Comment (bitscuit) [inline Makefile:81]: `RELEASE_VERSION` should replace `OPERATOR_VERSION`
```suggestion
BUNDLE_IMG ?= ibm-secretshare-operator-bundle:$(RELEASE_VERSION)
```

File: Makefile (hunk 1 above)
Comment (bitscuit) [reply Makefile:207]: yes, we should
```suggestion
	kustomize build config/manifests | operator-sdk generate bundle -q --overwrite --version $(RELEASE_VERSION) $(BUNDLE_METADATA_OPTS)
```

**Review by bitscuit — CHANGES_REQUESTED** [review]

**Comment by horis233** [comment]
@Daniel-Fan @bitscuit Nice catch! updated to use `release-version`

**Review by bitscuit — APPROVED** [review]
/lgtm

**Pull request merged**